rfd/0057-automatic-aws-server-discovery.md

```diff
@@ -235,6 +235,153 @@ discovery_service:
     teleport.dev/origin: "cloud"
 ```
 
+
+### Agentless installation
+
+In addition to supporting automatic Teleport agent installation, an agentless option
+will also be supported. This mode will update the OpenSSH CA to use the Teleport CA
+without installing the full Teleport Agent.
+
+The teleport ca and host keys will be generated and then uploaded to
+the the AWS Secret store where, for each node discovered a new secret
+will be created, each secret will have the following contents
+
+```json
+{
+	"ca_key": "...",
+	"host_key": "...",
+	"host_cert": "...",
+}
+```
+
+This will require changes to Teleport in order to support RBAC of OpenSSH based
```

Comment:
this mentions some RBAC changes, can you specify what exact changes are required

Reply:
This is referring to the proper OpenSSH servers "inventory management" - we want to make sure that you can register OpenSSH servers as "nodes" in Teleport so they appear in listings (e.g. tsh ls, web UI) and that RBAC works properly with them when they're registered this way. @capnspacehook is working on this (https://github.com/gravitational/teleport/issues/18627) and agentless EC2 discovery will be able to take advantage of this feature as well.

Reply:
I'll be submitting an RFD for changes related to #18627 in the next few days